msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/ClaimsTest.java

```diff
@@ -30,30 +31,25 @@ void testClaimsRequest_Format() {
         cr.requestClaimInIdToken("sub", new RequestedClaimAdditionalInfo(true, "248289761001", null));
         cr.requestClaimInIdToken("auth_time", new RequestedClaimAdditionalInfo(false, null, null));
 
-        assertEquals(cr.formatAsJSONString(), TestConfiguration.CLAIMS_REQUEST);
+        assertEquals(TestConfiguration.CLAIMS_REQUEST, cr.formatAsJSONString());
     }
 
     @Test
     void testClaimsRequest_MergeWithClientCapabilitiesAndClaimsChallenge() throws URISyntaxException {
 
-        List<String> values = new ArrayList<>();
-        values.add("urn:mace:incommon:iap:silver");
-        values.add("urn:mace:incommon:iap:bronze");
-
         ClaimsRequest cr = new ClaimsRequest();
-        cr.requestClaimInAccessToken("given_name", new RequestedClaimAdditionalInfo(true, null, null));
-        cr.requestClaimInAccessToken("email", null);
-        cr.requestClaimInIdToken("acr", new RequestedClaimAdditionalInfo(false, null, values));
-        cr.requestClaimInIdToken("sub", new RequestedClaimAdditionalInfo(true, "248289761001", null));
-        cr.requestClaimInIdToken("auth_time", new RequestedClaimAdditionalInfo(false, null, null));
+        cr.requestClaimInAccessToken("nbf", new RequestedClaimAdditionalInfo(true, "1701477303", null));
+
+        Set<String> capabilities = new HashSet<>();
+        capabilities.add("cp1");
 
         PublicClientApplication pca = PublicClientApplication.builder(
                 "client_id").
-                clientCapabilities(new HashSet<>(Collections.singletonList("llt"))).
+                clientCapabilities(capabilities).
                 build();
 
         InteractiveRequestParameters parameters = InteractiveRequestParameters.builder(new URI("http://localhost:8080"))
-                .claimsChallenge("{\"id_token\":{\"auth_time\":{\"essential\":true}},\"access_token\":{\"auth_time\":{\"essential\":true},\"xms_cc\":{\"values\":[\"abc\"]}}}")
+                .claimsChallenge(TestConfiguration.CLAIMS_CHALLENGE)
                 .claims(cr)
```

Comment:
Why both claims and claims channelge?

Can we deprecate this `claims` and ClaimsRequest object model?

Reply:
For simplicity I'm not deprecating in this PR/release, but I create this task to get it deprecated in the next one: https://github.com/AzureAD/microsoft-authentication-library-for-java/issues/855